fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_objectives.dm

```diff
@@ -66,17 +66,17 @@
 		"Chief Medical Officer",
 		"Quartermaster",
 	)
-	var/list/departs = list(
-		"Captain",
-		"Head of Security",
-		"Head of Personnel",
-		"Research Director",
-		"Chief Engineer",
-		"Chief Medical Officer",
+	// Captain doesn't go here because captain is head of the command department, and command has its own objective
+	var/list/heads_w_depart_index = list(
+		"Security" = "Head of Security",
+		"Service or Cargo" = "Head of Personnel",
```

Comment:
What the fuck, not service.

Reply:
Wouldn't the service department be included anyways because the code that checks for a valid job bases it on the